packages/expandable-card/src/ExpandableCard/ExpandableCard.types.ts

```diff
@@ -7,12 +7,12 @@ import { DarkModeProps } from '@leafygreen-ui/lib';
  * Types
  */
 export interface ExpandableCardProps
-  extends Omit<CardProps, 'contentStyle'>,
+  extends Omit<CardProps, 'contentStyle' | 'title'>,
     DarkModeProps {
   /**
    * The title of the card
    */
-  title: string;
+  title: React.ReactNode;
```

Comment:
do we need to make this `string | React.ReactNode`? otherwise, doesn't this become a breaking change?

Reply:
nit: would make this `ReactNode` instead of ` React.ReactNode` since it's already imported for the `description` prop

Reply:
But I like that nit, will update :)

Reply:
> React.ReactNode includes strings so should be fine
I'll see myself out 😶‍🌫️

Reply:
hahahahhaha